src/chrome/chromeDebugAdapter.ts

```diff
@@ -1203,7 +1203,19 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
             return;
         }
 
-        const committedBps = this._committedBreakpointsByUrl.get(script.url) || [];
+        // If we cannot find the script url in the committed breakpoints, we check for the canonicalized path. If found, use those committed breakpoints
+        // If we cannot find either the given url or the canonicalized url (or they are the same and the script simply cannot be found), committed breakpoints is an empty array
+        let committedBps = this._committedBreakpointsByUrl.get(script.url);
```

Comment:
If the scripts inside are canonicalized we should be able to just search for the canonicalized version